src/core/componentSystems/StatsSystem.js

```diff
@@ -0,0 +1,59 @@
+import { mrjsUtils } from 'mrjs';
+import { MRSystem } from 'mrjs/core/MRSystem';
+import { MRStats } from 'mrjs/core/entities/MRStats';
+
+import Stats from 'stats.js';
+
+import { HTMLMesh } from 'three/addons/interactive/HTMLMesh.js';
+import { InteractiveGroup } from 'three/addons/interactive/InteractiveGroup.js';
+
+// TODO: JSDoc
+
+export class StatsSystem extends MRSystem {
+    constructor() {
+        super(false);
+        this.statsEntities = [];
+    }
+
+    update() {
+        for (const entity of this.statsEntities) {
+            if (entity.stats === null && this.app) {
+                entity.stats = new Stats();
+                entity.stats.showPanel(0); // 0: fps, 1: ms, 2: mb, 3+: custom
+                document.body.appendChild(entity.stats.dom);
+
+                // Note: InteractiveGroup API is refactored in the newer Three.js rev.
+                //       Don't forget to update the followings when upgrading Three.js
+                entity.interactiveGroup = new InteractiveGroup(this.app.renderer, this.app.camera);
+                entity.object3D.add(entity.interactiveGroup);
+
+                // Note: HTMLMesh seems to have a bug that image is not displayed at a correct position.
+                //       in Three.js r161 or older.
+                // TODO: Upgrade Three.js to fix

```

Comment:
@michaelthatsit @hanbollar What do you think of upgrading Three.js to the latest?